app/src/main/java/fr/free/nrw/commons/utils/CommonsDateUtil.java

```diff
@@ -20,4 +20,10 @@ public static SimpleDateFormat getIso8601DateFormatShort() {
         simpleDateFormat.setTimeZone(TimeZone.getTimeZone("UTC"));
         return simpleDateFormat;
     }
+
```

Comment:
Please add java docs

Reply:
Please add what does this return value means too